cmake/FindXaudio2.cmake

```
@@ -13,6 +13,6 @@ check_cxx_source_compiles("
     #define _WIN32_WINNT     _WIN32_WINNT_WIN8
 
     #include <xaudio2.h>
-    int main() { return 0; }"
+    int main() { printf(\"%s\n\", XAUDIO2_DLL_A); return 0; }"
```

```suggestion
    int main() { printf(\"%s\\\\n\", XAUDIO2_DLL_A); return 0; }"
```
and also a missing `#include <cstdio>` for `printf` declaration